com.unity.render-pipelines.high-definition/Documentation~/Feature-Comparison.md

```diff
@@ -306,7 +306,7 @@ Managing LOD in this way has two consequences:
 
 | **Feature**                  | **Built-in Render Pipeline** | **High Definition Render Pipeline (HDRP)** |
 | ------------------------ | ------------------------ | ------------------------------- |
-| **Speed Tree**           | Yes                      | In research                     |
+| **Speed Tree**           | Yes                      | SpeedTree8 only. Expect visual changes for correctness in future versions. |
```

Comment:
Does "future versions" refer to versions of HDRP, Unity, or SpeedTree?
Also, should be SpeedTree 8 with a space, according to the vendor website.

Reply:
Future versions of HD and Universal SpeedTree 8 shadergraphs. It's referring specifically to the calculation of normals using custom interpolators. It's kind of behind-the-scenes... do you think it's worth mentioning? This comment should be removed in the 21.2+ PR.

Reply:
Ok, based on what you've said I've revised that to:

Currently HDRP and URP only support SpeedTree 8. In a future version of Shader Graph, this functionality will be visually different because of changes to normal calculation with custom interpolators.

Reply:
This is the HD feature comparison page. Does it make sense to mention universal here?

In c29ee75 I wrote it as the following, what do you think?
SpeedTree 8 only. Expect visual changes to normals for correctness in 21.2.

Reply:
Oh right, it is the HDRP page. Although I thought the visual changes were coming in 22.1?

Reply:
> SpeedTree 8 only. Expect visual changes to normals for correctness in 21.2.
is fine, but my sense was that the visual change was not in scope for 21.2?

Reply:
Oh I see-- when I said 21.2+, I meant 21.2 and later will have the updated normals.

Reply:
I am a bit confused. 

What I understand is:
This feature is being introduced in 21.2.
The normals are being changed in 21.2.
Why then state that the normals will change when they are part of the same release?

I also thought the delay in this change to the normals was why this shader won't be the default in URP until 22.1?